conf/groups.conf.d/slurm.conf.example

```diff
@@ -12,6 +12,16 @@ all: sinfo -h -o "%N"
 list: sinfo -h -o "%R"
 reverse: sinfo -h -N -o "%R" -n $NODE
 
+#
+# SLURM node-reservation bindings
+#
+[slurmresv,sr]
+map: scontrol -o show reservation $GROUP | grep -Po 'Nodes=\K[^ ]+'
+all: scontrol -o show reservation | grep -Po 'Nodes=\K[^ ]+'
+list: scontrol -o show reservation | grep -Po 'ReservationName=\K[^ ]+'
+reverse: scontrol -o show node $NODE | grep -Po 'ReservationName=\K[^ ]+' 2>/dev/null || true
```

Comment:
unfortunately, it seems Slurm only shows 1 resa per node. If you let ClusterShell do the reverse automatically, it will handle the case where a node is in several resa gracefully.

Reply:
Ah that's true: I didn't think about overlapping reservations (having the same node part of multiple simultaneous reservations is probably not that common), but `scontrol show node` only seems to return one reservation in that case, indeed. I guess that's a bug? 
I reported https://bugs.schedmd.com/show_bug.cgi?id=19884, let's see what SchedMD has to say,

Reply:
Mhhh.... the field is named `ReservationName`, singular. I don't expect it to return more than 1 name. And even if they confirmed this is a bug, we don't want to wait for the fix to be broadly available. Better merging this PR without `reverse` and maybe change it in the future.

Reply:
> the field is named ReservationName, singular. I don't expect it to return more than 1 name. 

Aah but that's not the only one, far from it :smile: 
```
$ scontrol show job 46818246 | grep -Po Partition=[a-z,]+
Partition=normal,hns
```

> And even if they confirmed this is a bug, we don't want to wait for the fix to be broadly available. Better merging this PR without reverse and maybe change it in the future.

I agree: the bug is confirmed, but it will most likely not be fixed very soon, so I'll remove the `reverse` command for now. Although I think it would still be more efficient to use it, as I understand it, to minimize calls to Slurm commands (one call to `scontrol` with an explicit `reverse`, vs. two to compute the result from `map` and `list`).